client/cz-crbug-issues.html

```diff
@@ -21,20 +21,28 @@
 
   <script>
     Polymer({
-      is: "cz-issues",
+      is: "cz-crbug-issues",
       behaviors: [ChromeZBehaviors.AJAXBehavior],
 
       onResponse: function(data) {
         data = JSON.parse(data);
         if (data.items == undefined)
           return [];
-        return data.items.filter(a => a.ID !== "");
+        return new IssueList(data.items.filter(a => a.ID !== "").map(item => {
```

Comment:
issues-lib is supposed to be vendor agnostic code, I don't think it makes sense to put Crbug specific code in there. This function performs the transformation from Crbug issue data into generic issue data. Would you prefer it live in crbug-lib.js?